account_receipt_report/reports/payment_receipt_report.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="report_account_payment_receipt">
+        <t t-call="web.html_container">
+            <t t-foreach="docs" t-as="o">
+                <t t-call="web.basic_layout">
+                    <t t-set="o" t-value="o.with_context(lang=o.partner_id.lang)"/>
+                    <div class="page">
+                        <style type="text/css">
+                            .square {
+                            border:1px solid;
+                            min-height: 60px;
+                            }
+                            .highlight {
+                            font-weight: bold;
+                            }
+                            .row-content {
+                            padding: 5px;
```

Comment:
Would you mind indenting these blocks properly please?

Also, afaik, `<style>` tags are only respected when found in `<head>`. You have to move it there possibly.

Reply:
Wouldn't it be easier to create a less file and reference it on the report (also, a class for the report should be necessary)